README.adoc

```diff
@@ -437,6 +437,33 @@ executed:
 $ sudo ./target/debug/omicron-package uninstall
 ----
 
+=== Test Environment
+
+When we deploy, we're effectively creating a number of different zones
+for all the components that make up Omicron (Nexus, Clickhouse, Crucible, etc).
+Since all these services run in different zones they cannot communicate with
+each other (and Sled Agent in the global zone) via `localhost`. In practise,
+we'll assign addresses as per RFD 63 as well as incorporating DNS based
+service discovery.
+
+For the purposes of local development today, we specify some hardcoded IPv6
+unique local addresses in `fd00:1de::/16`:
+
+[options="header"]
+|===================================================================================================
+| Service                    | Endpoint                                                             
+| Sled Agent: Bootstrap      | `[::]:12346`                                                         
+| Sled Agent: Dropshot API   | `[fd00:1de::]:12345`                                                 
+| Cockroach DB               | `[fd00:1de::5]:32221`                                                
+| Oximeter                   | `[fd00:1de::6]:12223`                                                
+| Nexus: External API        | `[fd00:1de::7]:12220`                                                
+| Nexus: Internal API        | `[fd00:1de::7]:12221`                                                
+| Clickhouse                 | `[fd00:1de::8]:8123`                                                 
+| Crucible Downstairs        | `[fd00:1de::9]:32345`, `[fd00:1de::10]:32345`, `[fd00:1de::11]:32345`
+|===================================================================================================
```

Comment:
I'm okay with this landing for now, but as a heads up, this is clearly configuration specific.

I plan on moving most of the "configuration" files out of "smf", and into a configuration overlay directory hierarchy.
When that happens, hopefully it makes sense to move this text there.